Donation-platform/src/main/webapp/trendingScript.js

```diff
@@ -37,6 +39,7 @@ function queryServer() {
         const out = [];
         charities.forEach((charity) => {
           out.push(charity.name);
```

Comment:
aren't "name" and "tags" private instance variables in the Charity class? I think we need to use getter methods for these.

Reply:
this method accesses the fields of the JSON, not of the objects themselves. the charity object attributes are private, so those can't be directly accessed outside the class.

Reply:
Ok. LGTM.